File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.rb
# @param {Integer[]} nums
# @return {Integer}
def remove_duplicates(nums)
    i = j = 0
    while j < nums.length()
        if nums[i] != nums[j]
            i += 1
            nums[i] = nums[j]
        end
        j += 1
    end
    return i + 1
end

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    var i = 0;
    var j = 0;
    while(j < nums.length) {
        if(nums[i] != nums[j]) {
            nums[++i] = nums[j];
        }
        j++;
    }
    return i + 1;
};